evennia/web/webclient/static/webclient/js/webclient_gui.js

```diff
@@ -175,8 +175,11 @@ function onKeydown (event) {
     }
 
     if (code === 27) { // Escape key
```

Comment:
This seems unrelated to this PR?

Reply:
Yeah when I changed the branch from master to dev, it pulled in a few subsequent unrelated PRs? I have no idea what this is about.

Reply:
I think this might have been because you merged into a local `develop` branch that was not in sync with the upstream version, maybe?